src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs

```diff
@@ -73,6 +73,8 @@
                          workflow-name (first input-names)
                          workflowId (workflow "workflowId")]
                    (style/create-link {:text workflowId
+                                       :target "_blank"
+                                       :style {:color "-webkit-link" :textDecoration "underline"}
```

Comment:
This style is consistent-enough that it should be factored out so we're not violating DRY. Maybe we need a `style/create-external-link` that applies this style by default? No need to fix for this PR but worth following up with another.